BraveShared/tr.lproj/BraveShared.strings

```diff
@@ -0,0 +1,1353 @@
+
+/* The context is to praise the user that they did a good job, to keep it up. It is used in full sentence like: 'Way to go! You earned 40 BAT last month.' */
+"ntp.goodJob" = "Tebrikler!";
+
+/* No comment provided by engineer. */
+"ntp.hideSponsoredImages" = "Sponsorlu resimleri gizle";
+
+/* No comment provided by engineer. */
+"ntp.learnMoreAboutRewards" = "Brave Rewards hakkında daha fazla bilgi edinin";
+
+/* No comment provided by engineer. */
+"ntp.learnMoreAboutSI" = "Sponsorlu resimler hakkında daha fazla bilgi edinin";
+
+/* A setting to enable or disable the device's keyboard from opening automatically when creating a new tab. */
+"ntp.settingsAutoOpenKeyboard" = "Klavyeyi Otomatik Aç";
+
+/* A setting to enable or disable background images on the main screen. */
+"ntp.settingsBackgroundImages" = "Arka Plan Resimlerini Göster";
+
+/* A button that leads to a 'more' menu, letting users configure additional settings. */
+"ntp.settingsBackgroundImageSubMenu" = "Arka Plan Resimleri";
+
+/* A selection to let the users only see a set of predefined, default images on the background of the app. */
+"ntp.settingsDefaultImagesOnly" = "Yalnızca Varsayılan Görüntüler";
+
+/* A selection to let the users see sponsored image backgrounds when opening a new tab. */
+"ntp.settingsSponsoredImagesSelection" = "Sponsorlu Görseller";
+
+/* Title on settings page to adjust the primary home screen functionality. */
+"ntp.settingsTitle" = "Yeni Sekme Sayfası";
+
+/* A button title to show more bookmarks, that opens a new menu. */
+"ntp.showMoreFavorites" = "Daha Fazla Göster";
+
+/* No comment provided by engineer. */
+"ntp.sponsoredImageDescription" = "Sponsorlu bu görüntüyü görüntüleyerek içerik oluşturucuları ve yayıncıları destekliyorsunuz.";
+
+/* The placeholder says 'Terms of Service'. So full sentence goes like: 'By turning Rewards, you agree to the Terms of Service'. */
+"ntp.turnRewardsTos" = "Ödülleri açarak %@'nı kabul ediyorsunuz.";
+
+/* Context menu item for opening all folder bookmarks */
+"OpenAllBookmarks" = "Tümünü Aç (%i)";
+
+/* Button title for opening the Brave Rewards panel to settings */
+"OpenBraveRewardsSettings" = "Brave Ödülleri Ayarlarını Aç";
+
+/* Context menu for opening image in new tab */
+"OpenImageInNewTab" = "Resmi Yeni Sekmede Aç";
+
+/* Context menu option for opening a link in a new private tab */
+"OpenNewPrivateTabButtonTitle" = "Yeni Gizli Sekmede Aç";
+
+/* Context menu item for opening a link in a new tab */
+"OpenNewTabButtonTitle" = "Yeni Sekmede Aç";
+
+/* See http://mzl.la/1G7uHo7 */
+"OpenPhoneSettingsActionTitle" = "Ayarları Aç";
+
+/* Button title to that opens a website. */
+"OpenWebsite" = "Web Sitesini Aç";
+
+/* Section name for other privacy settings */
+"OtherPrivacySettingsSection" = "Diğer Gizlilik Ayarları";
+
+/* Section name for other settings. */
+"OtherSettingsSection" = "Diğer Ayarlar";
+
+/* Sync pair device settings section title */
+"Pair" = "Çift";
```

Comment:
the meaning is totally diffe
rent here, the existing translation has meaning like "a pair of shoes"
It should be 

"Pair" = "Eşleştir";